velox/functions/prestosql/CMakeLists.txt

```diff
@@ -49,7 +49,8 @@ add_library(
   URLFunctions.cpp
   VectorArithmetic.cpp
   VectorFunctions.cpp
-  WidthBucketArray.cpp)
+  WidthBucketArray.cpp
+  ZipArray.cpp)
```

Comment:
naming: ZipArray.cpp -> Zip.cpp

WidthBucketArray has Array suffix to differentiate between two different implementations.